Desktop/html/js/jaspwidgets.js

```diff
@@ -358,10 +358,10 @@ JASPWidgets.Exporter = {
 	},
 
 	getHeaderStyles: function (element, exportParams) {
-		if (exportParams.isFormatted())
-			return JASPWidgets.Exporter.getStyles(element, ["padding", "text-align", "margin", "display", "float", "vertical-align", "font-size", "font", "font-weight", "color"]);
-		else
-			return "";//JASPWidgets.Exporter.getStyles(element, ["display", "float"]);
+		//if (exportParams.isFormatted())
```

Comment:
Could you add some comments here, because I don't understand why the isFormatted is commented out.

Reply:
In fact, I don't fully understand why we keep `isFormatted` here, this will make exported html and copy-paste style inconsistent， I do this so that the style of the assignment and pasting into the document is consistent with the style we export.

Reply:
But still, I agree a comment there saying something like "I commented this out because it breaks X" is probably good.
I see you also commented it out here:
```
	getNoteStyles: function (element, exportParams) {
		// if (exportParams.isFormatted())
			return JASPWidgets.Exporter.getStyles(element, ["margin", "padding", "max-width", "min-width", "display", "font-size", "font-weight", "font", "color", "border-top-style", "border-top-width", "border-top-color", "border-bottom-style", "border-bottom-width", "border-bottom-color"]);
		// else
		// 	return JASPWidgets.Exporter.getStyles(element, ["max-width", "min-width"]);
	},
```

But the last changes before that were all from 8 years ago... 
Im not even sure where and how I would see the difference between using `isFormatted` or not